### ISSUE TITLE
Monitor Request Skipping

### ISSUE THREAD

**Issue by cool-george** [a85/POSTMan-Chrome-Extension#4251]
## App Details:

```
Postman for Mac
Version 4.11.0
OS X 16.4.0 / x64
```

## Issue Report:
Just monitor a collection every 15 minutes for about 8 hours. Skim through some of the Monitor Performance Chart and you should see about one out of every 8 scheduled requests have been skipped. I'm not sure if this is just a reporting issue or just a glitch on my part (shouldn't be since I barely have any code in the requests). I was also wondering if I was being charged by the requests showing or by the number of requests that are supposed to be showing.

**Comment by godfrzero** [comment]
You'll be charged based on the number of requests actually sent. What exactly do you mean when you say the request is skipping? If possible, please include screenshots of the performance graph and relevant logs. If these are things which you can't post publicly, send an email to help@getpostman.com and we can help you out there.

**Comment by cool-george** [comment]
![monitor skipping proof - copy](https://user-images.githubusercontent.com/35109255/36791010-a980a584-1c64-11e8-85b2-05e26b3dd8f7.PNG)

I hope this is what you mean. Also, no request, no logging of request stuffs. I know its hard to see but, there are a few places that should have had a request in between them.

**Comment by cool-george** [comment]
![monitor skipping proof2](https://user-images.githubusercontent.com/35109255/36865329-371c701e-1d5d-11e8-8433-d52d499f2801.PNG)

Here is another screenshot taken a few minutes ago. As you might have noticed, I've set the interval to 10 minutes and still there are some requests being skipped.

**Comment by godfrzero** [comment]
Looks like this has to do with higher than normal load on our systems. We're rolling out an improved scheduling algorithm over the next week, so these delays should be reduced by then. I'll post here once the updates are out.

**Comment by cool-george** [comment]
Great! Postman is a really great app, so I'd hate to move to something else. I actually stumbled across an old post in stackoverflow where the creator of the app asked some questions that I was asking. Some of the comments were newer and sounded like nostalgia. It piqued my interest so I clicked the Postman link and was surprised when I found Postman was EXACTLY what I was searching for, and it was free! Keep up the good job!
